setup.py

```diff
@@ -105,6 +106,13 @@ def find_dlib():
         "lib64": lib64
     }
 
+def clean_build_area(self):
```

Comment:
Not that this matters a great deal, but should this be a method of `custom_build_ext`, or is there some reason an external function makes more sense? The NVCC compiler function I guess could also be a method, though I'm not sure the extra effort to change it makes a significant difference. I'm fine with this either way, so feel free to go ahead with the current version.